indexer/src/main.rs

```diff
@@ -139,6 +141,27 @@ async fn handle_message(
     Ok(())
 }
 
+async fn download_genesis_file(opts: &configs::Opts) -> anyhow::Result<String> {
+    let chain = match opts.chain_id {
+        ChainId::Mainnet(_) => "mainnet",
+        ChainId::Testnet(_) => "testnet",
+    };
+
+    info!(
+        target: INDEXER_FOR_EXPLORER,
+        "Downloading {} genesis file", chain
+    );
+
+    let resp = reqwest::get(opts.genesis_file_url()).await?;
+
+    let file_path = format!("{}-genesis.json", chain);
```

Comment:
I don't think it's necessary to have a prefix with a chain id for the file. Usually, a single instance of an indexer is used for one of the chains.
The only thing we should care about is ensuring the file is stored next to the indexer binary.

Another variant is to follow `nearcore` approach when the configs and files required by the instance is stored in under the `--home-dir`.

Reply:
> I don't think it's necessary to have a prefix with a chain id for the file. Usually, a single instance of an indexer is used for one of the chains.

I've removed the chain ID prefix 👍🏽 

> The only thing we should care about is ensuring the file is stored next to the indexer binary

Updated so `genesis.json` is stored next to the executable. During local dev that means storing in`./target/{debug|release}`.